scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -286,6 +294,12 @@ public Thread newThread(Runnable r) {
 				return ret;
 			}
 		});
+		if (handshakerTaskExecutor == null) {
+			handshakerTaskExecutor = Executors.newFixedThreadPool(
+					Runtime.getRuntime().availableProcessors(),
+					new DaemonThreadFactory("DTLS Handshaker Task", SCANDIUM_THREAD_GROUP));
```

Comment:
I guess because we do not want the JVM to have to wait until all handshaker tasks have finished before being able to shut down, or do we?